README.md

```diff
@@ -1 +1,34 @@
-# mona
\ No newline at end of file
+# mona
+
+## Client application
+
+### Prerequisites
+
+* `nodejs 8.4.0`
+* `npm 5.5.1`
+
+**Note** You probably shouldn't care of the versions. I wrote these here because
```

Comment:
care about?.. Just spend 10 minutes googling and can't figure the right preposition. Maybe just say 'shouldn't worry about the versions'?

Reply:
Yeah, sounds better!